bin/pasta_show_maintainers.py

```diff
@@ -0,0 +1,299 @@
+            for key, value in containing_dirs_filt.items()}
+        containing_dirs_filt = {key:reduce_dirs(value) 
+            for key, value in containing_dirs_filt.items()}        
+    # Sort subsystems by decreasing total line counts
+    return decreasing_order(counts), containing_dirs, containing_dirs_filt
+
+def get_counts_by_maintainer(all_maintainers, file_sizes, 
+                                verbose, filter_by_files = None):
+    maintainer_to_subsystem = dict()
+    # maps maintainers to total file sizes they are responsible for
+    counts = dict()
+    for filename in file_sizes:
+        subsystems = all_maintainers.get_subsystems_by_file(filename)
+        for subsystem in subsystems:
+            maintainers = all_maintainers.get_maintainers(subsystem)
+            for maintainer in flatten(maintainers):
+                if not (type(maintainer) == str 
+                        or maintainer[0] == '' 
+                        or maintainer[1] == ''):
+                    name = pretty_name(maintainer)
+                    loc, byte = file_sizes[filename]
+                    if name in counts:
+                        maintainer_to_subsystem[name].add(subsystem)
+                        counts[name].increase(filename, loc, byte,
+                                filter_by_files and filename in filter_by_files)
+                    else:
+                        maintainer_to_subsystem[name] = set([subsystem])
+                        counts[name] = CountItem(filename, loc, byte,
+                                filter_by_files and filename in filter_by_files)
+                elif verbose:
+                    log.info('Ignoring mailing list:'+ str(maintainer))
+    if filter_by_files:
+        # If filter is given, only return the relevant lines:
+        counts = {name:item for name,item in counts.items() 
+                    if item.display_in_filter()}      
+    # Sort maintainers by decreasing total line counts
+    return decreasing_order(counts), maintainer_to_subsystem
+
+def show_maintainers(config, sub, argv):
+    parser = ArgumentParser(description='Display file sizes grouped by ' 
+                                            'maintainers or subsystems')
+    parser.add_argument('show_maintainers', nargs=1, type=bool)
+    parser.add_argument('--entries', action='store_true', 
+                            help='groups by subsystems')
+    parser.add_argument('--verbose', action='store_true', 
+                            help='Show ignored exceptions on the way')
+    parser.add_argument('--smallstat', action='store_true', help='Simple view')
+    parser.add_argument('--bytes', action='store_true', help='Show byte counts')
+    parser.add_argument('--outfile', type=str, help='Output to a csv file')
+    parser.add_argument('--file', type=str, help='Specify MAINTAINERS file')
+    parser.add_argument('--filter', type=str, help='Filter by file list')
```

Comment:
Please try to improve the description of this argument. I had to walk through the code in order to understand, that --filter needs to point to the defconfig.files file.